stories/SQFormScrollableCardsMenuWrapper.stories.js

```diff
@@ -43,35 +43,13 @@ function ScrollableDetails() {
     actions.resetForm();
   };
 
-  /**
```

Comment:
these changes are ancillary, and after putting it back into SQFormScrollableCard, are unnecessary here